src/components/messages/Message/Message.scss

```diff
@@ -13,20 +59,30 @@
     max-width: 50%;
     overflow-wrap: break-word;
     padding: 0.4em 0.8em;
-    margin-top: 0.5em;
+    will-change: border-radius;
+    transition: border-radius ease 0.2s;
   }
 
   &__from-peer &__bubble {
     background: var(--sender-bubble-bg);
     box-shadow: var(--sender-box-shadow);
-    border-radius: 16px 16px 16px 6px;
     color: var(--fg-color-1);
   }
 
   &__from-sender &__bubble {
     box-shadow: inset 1px 1px 4px #99cbff, inset -1px -1px 4px #1464b8;
     background: #3396ff;
-    border-radius: 16px 16px 6px 16px;
     color: #ffffff;
   }
 }
+
+@keyframes bounce-up {
```

Comment:
Sorry for the lack of context on the issues but the animation should be a scaling animation.
You can use [this](https://github.com/WalletConnect/minisite-web3inbox/blob/e5aa0dc9bbc9f5803b53c60052bed6ecf1a6d4f4/src/lib/components/messages/SenderBubble.tsx#L22) as a reference